### ISSUE TITLE
Missing cover hashes from old filename scheme

### ISSUE THREAD

**Issue by ghost** [noaione/cmx-ermitteln#2]
The current collection of cover hashes is missing an unknown number of covers which use an old filename scheme from prior to and during the switch to Amazon as a CDN. I personally consider this low priority for a few reasons:
- If found, implementing these hashes would probably require some reworking of ermitteln since the URL scheme is not as simple as what is currently assumed.
- These covers are seemingly only for some books released pre-2016, making them less likely to be both currently needed and unavailable elsewhere.

Thought a public issue might be a good idea since I have no current plans to research this myself beyond what is below.

<br>

**Wayback Machine, October 2014**
`Infinite Crisis: Fight for the Multiverse (2014) #23`
`https://dcomixologyssl.sslcs.cdngc.net/c/DIG029449_1.jpg` (dead link)
ASIN: `B00OACHT2G`
Release Date (per listing): `October 19, 2014`

**Wayback Machine, October 2015**
`Wonder Woman '77 (2015) #8`
`https://m.media-amazon.com/images/S/cmx-images-prod/Item/262616/DIG063146_1.jpg` (working link)
ASIN: `B013VW0E0Y`
Release Date (per listing): `September 3, 2015`

**To summarize some basic info:**
- I'm not sure what the numbers in the filename signify, if anything.
- The numbers in the filename are rather large, making iterating impractical (on top of having to iterate through cmx ids to get the folder name, which is a separate issue)
- The currently assumed URL scheme of `.../Item/<ID>/<ID>.jpg` (e.g. `.../Item/262616/262616.jpg`) returns `Not found`.

You can find plenty more URL examples on the Wayback Machine if this is something you're interested in.
